views/pages/resources/cheque-requests.pug

```diff
@@ -0,0 +1,38 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+    link(href="/assets/css/pages/resources/cheque-requests.css" rel="stylesheet")
+append scripts
+    script(src="/assets/js/collapsible.js" defer)
+
+block body
+    +banner-image("/assets/img/banners/mathsoc-wall.jpeg")
+    div(class="content")
+        section
+            h1 #{data.title}
+            each form in data.formLinks
+                a(href=form.link target="_blank" class="form-link")

```

Comment:
Thoughts on making these visually look like buttons instead of the text-based links from the wordpress site?

(quick MS paint sketch below: )

<img width="400px" src="https://user-images.githubusercontent.com/37753525/222322352-550bdb69-2de0-45cd-8291-df00b5d3ac18.png"/>

Reply:
I feel they're a little difficult to read, and might get chunky if we add more forms in the future. But otherwise I don't have a strong opinion and I can add them if that's what we want for design